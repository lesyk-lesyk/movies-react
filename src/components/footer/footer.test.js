import React from 'react';
import { Footer } from './index';
import renderer from 'react-test-renderer';

describe('Footer', () => {
  it('renders', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
