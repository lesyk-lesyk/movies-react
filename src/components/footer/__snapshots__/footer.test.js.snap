// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<footer
  className="main-footer"
>
  <div
    className="container"
  >
    <span
      className="site-name"
    >
      netflixroulette
    </span>
  </div>
</footer>
`;
